example/view/CMakeLists.txt

```diff
@@ -1,3 +1,10 @@
-set(CMAKE_FOLDER "${CMAKE_FOLDER}/algorithm")
+set(CMAKE_FOLDER "${CMAKE_FOLDER}/view")
 
 rv3_add_test(example.view.transform example.view.transform transform.cpp)
+
+find_program(BASH_PROGRAM bash)
+
+if (BASH_PROGRAM)
+  add_test(example.view.check_example_outputs
+          ${BASH_PROGRAM} ${CMAKE_CURRENT_SOURCE_DIR}/check_example_outputs.sh ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
you can use the cross platform `cmake -E compare_files`. See https://cmake.org/cmake/help/latest/manual/cmake.1.html#run-a-command-line-tool

Reply:
I updated the code to use compare_files. However, I couldn't figure out how to write the test output file without using `sh`. If `sh` is not found, it should skip the test and the doc build should work fine since it uses the checked in golden.

Let me know if you have any suggestions, I'm not familiar with cmake.

Reply:
you can use [PASS_REGULAR_EXPRESSION](https://cmake.org/cmake/help/latest/prop_test/PASS_REGULAR_EXPRESSION.html) property and omit the reference output as follows:
```cmake
rv3_add_test(example.view.transform example.view.transform transform.cpp)
set_tests_properties(range.v3.example.view.transform PROPERTIES PASS_REGULAR_EXPRESSION "\\[0.5,1,1.5\\]")
```

Reply:
Done.